### PR TITLE
Add curl health checks for ecomm, studio and LMS to Travis

### PR DESCRIPTION
**Pull request by jbarciauskas**
Create 'make healthchecks' to run healthchecks against active services. Still need to add credentials (not currently running on my machine for some reason) and discovery (once it's been fixed).

### CODE REVIEW

**Comment by jbarciauskas** [comment]
fyi @edx/docker-devstack-working-group

**Review by gsong — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -55,3 +55,6 @@ lms-shell: ## Run a shell on the LMS container
 
 studio-shell: ## Run a shell on the Studio container
 	docker exec -it edx.devstack.studio env TERM=$(TERM) /edx/app/edxapp/devstack.sh open
+
+healthchecks:
```
Comment (clintonb) [inline Makefile]: Add help text explaining what this does.

**Review by clintonb — APPROVED** [review]

File: healthchecks.sh [hunk 2]
```diff
@@ -0,0 +1,11 @@
+set -e
+set -x
+
+echo "Checking LMS heartbeat:"
+curl http://localhost:18000/heartbeat #LMS
+echo
+echo "Checking Studio heartbeat:"
+curl http://localhost:18010/heartbeat #LMS
```
Comment (doctoryes) [inline healthchecks.sh]: Nit: Maybe use `jq` to check the heartbeat results? Or perhaps just check the HTTP response code?

Comment (jbarciauskas) [reply healthchecks.sh]: This is relying on the fact that curl exits with a non-zero exit code if the response from the server is not a 200 response code. Is that good enough or do we want something more? I figured exiting with an error + visual inspection was good enough.

Comment (doctoryes) [reply healthchecks.sh]: That's enough. 👍

**Pull request merged**